### ISSUE TITLE
Always open Slack in New Window

### ISSUE THREAD

**Issue by raalarcon** [wavebox/waveboxapp#1029]
* Wavebox Version: 4.10.1
* Operating System & Version: Windows 10 Pro v 1809

**Is your feature request related to a problem? Please describe.**
No Problem, just interested in optional default functionality

**Is your feature request related to an account type?**
Slack

**Describe the solution you'd like**
This request may arise from how I personally prefer to work with Slack, but hopefully others could benefit as well, regardless of the account.
I always start my day opening Wavebox, however I prefer that Slack be in a separate (new) window, so I right click on the icon, and select open in new window.  I wonder if this can be enabled as a default behavior when clicking the icon, or even upon start.

**Additional info**
Regarding Notifications, when Slack is running in a new window, whenever I receive a Windows notification pop-up, if I click, it opens a new session in Slack, not to the already open window.

**Comment by Thomas101** [comment]
Yeah, this is a fair point. The open in a new window really is a bit of a hack in terms of usability. The app doesn't make the link that the opened window is actually the proper service more of just another generic window.

I've got a bunch of work planned which touches on the way accounts work so it would be a prime time to roll this in :)
